apps/src/code-studio/initApp/loadApp.js

```diff
@@ -1,17 +1,203 @@
-/* global dashboard, appOptions, addToHome */
+/* global dashboard, appOptions, addToHome script_path, CDOSounds, appOptions, trackEvent, Applab, Blockly */
```

Comment:
Needing to point out these globals code smell :nose: that will be worth visiting in the future :+1:.  For now, do we need a comma after addToHome?  If not, maybe there shouldn't be any commas here?